src/main/java/be/syntra/devshop/DevshopFront/security/CustomLogoutHandler.java

```diff
@@ -0,0 +1,30 @@
+package be.syntra.devshop.DevshopFront.security;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.web.authentication.logout.LogoutSuccessHandler;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class CustomLogoutHandler implements LogoutSuccessHandler {
+
+    private Logger logger = LoggerFactory.getLogger(CustomLogoutHandler.class);
```

Comment:
I tried logging with annotated  @Slf4j in other parts and it should work

Reply:
good to know. Won't refactor untill fixing tests